### DISCUSSION TITLE
Move from zwave js ui home assistant to truenas scale

### DISCUSSION THREAD

**Discussion by daschmidt1994** [zwave-js/zwave-js-ui#3049] [Q&A] [answered]
Is it possible, to switch from the homeassistant addon zwave js ui to the truecharts app without
to include all devices new?

**Comment by cornellrwilliams** [comment, marked as answer]
Yes I believe you can just make a backup of the ZWaveJSUI store files then reimport the files on the new machine. The store filea contain all of your settings, backups, and so on. The node ids get stored in the nvm of the controller.

**Reply by robertsLando** [reply]
I confirm what @cornellrwilliams said. Just do a store backup and restore it on the new instance and you are done. An NVM backup is not necessary but you can do it (no  need to restore it BTW if everything goes well).

**Reply by daschmidt1994** [reply]
thanks, will try it in a few days.
I will switch from the zwavejsui addonbfrom home assistant to the addonbon my truenas scale with truecharts

**Reply by robertsLando** [reply]
The store backup can be done directly from store explorer tab, just press on the fab and select backup, it will download a zip file with all the files needed (not everything on store needs to be backuped). Then in the new instance go on store explorer, select restore and use the previous zip file to restore it
